database.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+let mongoServer;
+const uri="mongodb+srv://sckchcm:17%40$aksham@cluster0.rpxpqex.mongodb.net/?retryWrites=true&w=majority"
+
+const startDatabase = async () => {
+  await mongoose.connect(uri)
```

Comment:
Handle potential errors during the database connection. This can be done by wrapping the `mongoose.connect` and `mongoose.disconnect` calls in try-catch blocks. [important]